pkg/apis/tenancy/v1alpha1/helper/helper.go

```diff
@@ -26,6 +26,13 @@ import (
 	"github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
 )
 
+const (
+	WorkspaceNameLabel = "workspaces.kcp.dev/name"
+)
```

Comment:
This feels significant enough to me to live in the `v1beta1` package, perhaps alongside the type definitions, but it appears there is a pattern of defining some helpers here so I opted to drop it in here for the time-being. More than happy to move it as desired.